src/toil_scripts/lib/files.py

```diff
@@ -78,3 +135,42 @@ def consolidate_tarballs_job(job, fname_to_id):
                         tarinfo.name = os.path.join(output_name, fname, os.path.basename(tarinfo.name))
                         f_out.addfile(tarinfo, fileobj=f_in_file)
     return job.fileStore.writeGlobalFile(out_tar)
+
+
+def get_files_from_filestore(job, work_dir, input_dict):
```

Comment:
As I said earlier, I think the unpacking tarfiles is not a good behavior to have.